### ISSUE TITLE
Cannot deploy helm in kubernetes 1.16.8

### ISSUE THREAD

**Issue by willgraf** [vanvalenlab/kiosk-console#345]
**Describe the bug**
When creating a cluster with `KUBERNETES_VERSION` 1.16, the `gke/deploy/helm` fails with the following error:

```
Deploying kubernetes resources...
serviceaccount/tiller created
clusterrolebinding.rbac.authorization.k8s.io/tiller-cluster-rule created
$HELM_HOME has been configured at /var/lib/helm.
Error: error installing: the server could not find the requested resource
make[1]: [tasks/Makefile.gke:236: gke/deploy/helm] Error 1 (ignored)
```

**To Reproduce**
Set `KUBERNETES_VERSION` to `1.16` and create a cluster.

**Expected behavior**
Helm should install properly.

**Additional context**
It looks like this is a [documented issue](https://github.com/helm/helm/issues/6374). We are also stuck using helm2. A migration to using helm3 may resolve this issue, and can be done by upgrading the base `cloudposse/geodesic` base image.

**Comment by willgraf** [comment]
Updating to the latest geodesic base image (and therefore helm3) does seem to have fixed this issue.

**Comment by willgraf** [comment]
Closed by #370
